source/guide_coturn.rst

```diff
@@ -92,22 +94,22 @@ Replace values in brackets ``<value>`` with your values.
  listening-ip=0.0.0.0
  relay-ip=::
  relay-ip=0.0.0.0
+ external-ip=<IP_OF_YOUR_UBERSPACE>
```

Comment:
which IP address? the external one, e.g. `185.26.156.62` for `tuttle.uberspace.de`? Please add a comment or hint somewhere on how to get that IP.